bin/get_image_config.py

```diff
@@ -97,6 +97,7 @@ def process_tags(setup, data, images):
     images[-1]['IMAGE_TAG']=get_key('tag', img_data)
     images[-1]['IMAGE_TAG_ALIAS']=get_key('alias', img_data)
     images[-1]['MULTI_ARCH']=get_key('multi_arch', img_data)
+    images[-1]['BASE_UPSTREAM_CHECKSUM']=from_manifest
```

Comment:
`from_manifest` points to base image e.g. for cmssw/el8:grid the `from_manifest` will be `cmssw/el8:runtime` ... right?

Reply:
I am looking into it

Reply:
Ah, yes! `from_manifest` points to the base image in each case. Then in https://github.com/aandvalenzuela/cms-docker/blob/track-base/bin/docker-images-has-changed.py#L44 we construct a dictionary only with the checksum values of the `almalinux` images and we add the corresponding `almalinux` base checksum to the property file depending on the arch